lib/qless/job.rb

```diff
@@ -110,6 +111,19 @@ def self.build(client, klass, attributes = {})
       new(client, attributes)
     end
 
+    # Converts a hash of job options (as returned by job.to_hash) into the array
+    # format the qless api expects.
+    def self.build_opts_array(opts)
+      result = []
+      result << JSON.generate(opts.fetch(:data, {}))
+      result.concat([opts.fetch(:delay, 0)])
+      result.concat(['priority', opts.fetch(:priority, 0)])
+      result.concat(['tags', JSON.generate(opts.fetch(:tags, []))])
+      result.concat(['retries', opts.fetch(:retries, 5)])
+      result.concat(['depends', JSON.generate(opts.fetch(:depends, []))])
```

Comment:
YMMV, but the benefit of `Hash#fetch` comes in when you expect the value in the Hash to be `false` or `nil` sometimes, otherwise, might as well do `opts[:depends] || []`

@jzaleski, I miss you :kiss: